deployments/base/aero/configuration.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Compound AERO",
+  "symbol": "cAEROv3",
+  "baseToken": "AERO",
+  "baseTokenAddress": "0x940181a94A35A4569E4529A3CDfB74e38FD98631",
+  "borrowMin": "1e18",
```

Comment:
1 ARRO = 1.16$, thus min should be higher

Reply:
It does not matter. It shows the minimum borrow amount